invenio_records_rest/serializers/marshmallow.py

```diff
@@ -29,25 +29,28 @@
 from .base import PreprocessorMixin
 
 
-class MarshmallowSerializer(PreprocessorMixin):
```

Comment:
I would keep ``MarshmallowSerializer`` to avoid nasty surprises that it has been removed (i.e. like you have ``JSONLDSerializer``).